walkthroughs/eks-canary-deployments-pipeline/microservices/yelb-appserver/specfiles/kubernetes.yml

```diff
@@ -0,0 +1,149 @@
+---
+apiVersion: v1
+kind: Namespace
+metadata:
+  labels:
+    mesh: yelb
+    appmesh.k8s.aws/sidecarInjectorWebhook: enabled
+  name: yelb
+---
+apiVersion: appmesh.k8s.aws/v1beta2
+kind: Mesh
+metadata:
+  name: yelb
+spec:
+  namespaceSelector:
+    matchLabels:
+      mesh: yelb
+---
+apiVersion: appmesh.k8s.aws/v1beta2
+kind: VirtualNode
+metadata:
+  name: yelb-appserver-NEW
```

Comment:
What's the reason for the `NEW` naming on the resources in this file?

Reply:
Renamed CANARY_DEPLOYMENT to CANARY_ROUTES and NEW to CANARY_VERSION in commit [1aafa19](https://github.com/aws/aws-app-mesh-examples/pull/348/commits/1aafa195e9a53620d0cba6ebd23d3fb9844ee32a)